src/api/platform/account/create_user_key.rs

```diff
@@ -1,43 +1,34 @@
 use async_trait::async_trait;
+
 use reqwest::{Method, RequestBuilder};
 use serde::Serialize;
 
 use crate::api::client::{ApiError, ApiRequest, PlatformApiRequest};
-use crate::api::platform::ApiKey;
-use crate::codec::crypto::VerifyingKey;
+use crate::api::platform::ApiUserKey;
 
 // todo(sstelfox): currently we support registering any key with permissions to the account, then
 // that key gets full access to the account and any encrypted data even if the user doesn't have
 // access. We need to adjust our permission model to restrict the key usage to the buckets it has
 // permission over. We'll need a workflow to upgrade keys to the root of the account for things
 // like managing billing.
 #[derive(Serialize)]
-pub struct RegisterApiKey {
-    #[serde(skip)]
-    fingerprint: String,
-
-    public_key: String,
+pub struct CreateUserKey {
+    name: String,
+    public_key_pem: String,
```

Comment:
The `_pem` suffix isn't a particularly useful change and definitely a term I want to get out of our names altogether. The format may be useful to be in the documentation but names should describe the purpose of the thing. We wouldn't name variables `public_key_json` which is the equivalent of that this is.